src/pyOpenMS/pxds/LPWrapper.pxd

```diff
@@ -7,48 +7,74 @@ cdef extern from "<OpenMS/DATASTRUCTURES/LPWrapper.h>" namespace "OpenMS":
+        void setObjectiveSense(Sense sense) nogil except + # wrap-doc:Set objective direction
         Sense getObjectiveSense() nogil except +
-        Int getNumberOfColumns() nogil except +
-        Int getNumberOfRows() nogil except +
-        void setElement(Int row_index, Int column_index, double value) nogil except +
-        double getElement(Int row_index, Int column_index) nogil except +
+        Int getNumberOfColumns() nogil except + # wrap-doc:Get number of columns
+        Int getNumberOfRows() nogil except + # wrap-doc:Get number of rows
+        void setElement(Int row_index, Int column_index, double value) nogil except +# TODO
+        double getElement(Int row_index, Int column_index) nogil except +# TODO
         void readProblem(String filename, String format_) nogil except +
+            # wrap-doc:
+                #   Read LP from file
+                #   -----
+                #   :param filename: Filename where to store the LP problem
+                #   :param format: LP, MPS or GLPK
+
         void writeProblem(const String & filename, WriteFormat format_) nogil except +
+            # wrap-doc:
+                #   Write LP formulation to a file
+                #   -----
+                #   :param filename: output filename, if the filename ends with '.gz' it will be compressed
+                #   :param format: MPS-format is supported by GLPK and COIN-OR; LP and GLPK-formats only by GLPK
+
         Int solve(SolverParam & solver_param, Size verbose_level) nogil except +
+            # wrap-doc:
+                #   solve problems, parameters like enabled heuristics can be given via solver_param
+                #   -----
+                #   The verbose level (0,1,2) determines if the solver prints status messages and internals.
+                #   -----
+                #   :param solver_param:
+                #   :param verbose_level:
+                #   :returns: solver dependent 
+
         SolverStatus getStatus() nogil except +
+            # wrap-doc:
+                #   Get solution status
+                #   -----
+                #   :returns: status: 1 - undefined, 2 - integer optimal, 3- integer feasible (no optimality proven), 4- no integer feasible solution
+
         double getObjectiveValue() nogil except +
         double getColumnValue(Int index) nogil except +
         Int getNumberOfNonZeroEntriesInRow(Int idx) nogil except +
         void getMatrixRow(Int idx, libcpp_vector[ int ] & indexes) nogil except +
-        SOLVER getSolver() nogil except +
+        SOLVER getSolver() nogil except + # wrap-doc:Get currently active solver
 
 cdef extern from "<OpenMS/DATASTRUCTURES/LPWrapper.h>" namespace "OpenMS::LPWrapper":
     
     cdef cppclass SolverParam "OpenMS::LPWrapper::SolverParam":
-        SolverParam() nogil except +
+        SolverParam() nogil except + # wrap-doc:Struct that holds the parameters of the LP solver
```

Comment:
Would remove Struct, since it is not visible if it is a Struct or Class in pyopenms. 
e.g. Hold the parameters of the LP solver